R/discover_rules.R

```diff
@@ -1,23 +1,24 @@
 #' @title
-#' Generate causal rules
+#' Discover Rules
 #'
 #' @description
-#' Generates causal decision rules.
+#' Discover the minimal set of Rules Linearly Decomposing the Conditional
+#' Average Treatment Effect (CATE).
 #'
 #' @param X A covariate matrix.
 #' @param ite An estimated ITE.
 #' @param method_params A vector of method parameters.
 #' @param hyper_params A vector of hyper parameters.
 #'
 #' @return
-#' A vector of causal rules.
+#' The minimal set of Rules Linearly Decomposing the CATE.
```

Comment:
A minimal set of rules linearly decomposing the Conditional Treatment Effect (CATE).

Reply:
addressed in the new PR